.github/workflows/instrumented-tests.yml

```
@@ -78,13 +78,13 @@ jobs:
       # Configure Gradle for optimal use in GitHub Actions, including caching of downloaded dependencies.
       # See: https://github.com/gradle/actions/blob/main/setup-gradle/README.md
       - name: Setup Gradle
-        uses: gradle/actions/setup-gradle@5e2ebd065dc2488b7a6ad670704656cbbe1e8f60 # v6.1.1
+        uses: gradle/actions/setup-gradle@50e97c2cd7a37755bbfafc9c5b7cafaece252f6e # v6.1.0
 
       - name: Run tests on API ${{ matrix.api-level }}
         uses: reactivecircus/android-emulator-runner@e89f39f1abbbd05b1113a29cf4db69e7540cae5a # v2.37.0
         with:
           api-level: ${{ matrix.api-level }}
-          target: ${{ matrix.api-level >= 28 && 'playstore' || 'default' }}
+          target: ${{ matrix.api_level >= 35 && 'google_apis_ps16k' || matrix.api-level >= 28 && 'google_apis' || 'default' }}
```

actionlint (1.7.12)

[error] property "api_level" is not defined in object type {api-level: number; orientation: {name: string; rotation: number; skin: string}}

(expression)